loadtest/control/simplecontroller/actions.go

```diff
@@ -159,23 +159,24 @@ func (c *SimpleController) createGroupChannel() control.UserStatus {
 }
 
 func (c *SimpleController) viewChannel() control.UserStatus {
-	return c.newErrorStatus(errors.New("not implemented"))
-	/*
-		channel, err := c.user.Store().Channel("") // TODO: fetch channel randomly?
-		if err != nil {
-			return control.UserStatus{User: c.user, Err: err, Code: user.STATUS_ERROR}
-		}
+	team, err := c.user.Store().RandomTeam()
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	channel, err := c.user.Store().RandomChannel(team.Id)
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
```

Comment:
In a smarter controller this should become a weighted random selection since not all channels have the same probability of being chosen. But for this case a simple random pick is good enough to stress the API.